.github/workflows/unit-test-php.yml

```
@@ -0,0 +1,119 @@
+name: PHPunit with database
+
+on:
+  workflow_call:
+    inputs:
+      versions:
+        default: >-
+          ["8.1"]
```

Default to current and future as we are working towards 8.2
```suggestion
          ["8.1","8.2"]
```